.github/workflows/test-scripts.yml

```diff
@@ -12,6 +12,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - run: |
+        sudo apt-get install conntrack
```

Comment:
Does this get stuck on interactive prompts without `-y`?

Reply:
An example I saw didn't, but to be safe I updated the script to fire when it's changed so we can know for sure.

Reply:
Might be covered by the runner VM: https://github.com/actions/virtual-environments/commit/b1026148e82a01ca6f1a9e3157ad806ec825df46